### ISSUE TITLE
Failed: phase compiling: libijs1-shlibs-0.35-1 failed

### ISSUE THREAD

**Issue by dak180** [fink/fink-distributions#964]
```
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking build system type... configure: error: /bin/sh ./config.sub -apple-darwin21.6.0 failed
### execution of ./configure failed, exit code 1
### execution of /tmp/fink.YXu5P failed, exit code 1
```

```
Package manager version: 0.45.99.git
Distribution version: selfupdate-rsync Mon Jan 30 11:20:25 2023, 12.0, arm64
Trees: local/main remote/dak180 stable/main local/injected
Xcode.app: 14.2
Xcode command-line tools: 14.2.0.0.1.1668646533
Max. Fink build jobs:  1
```

**Comment by dak180** [comment]
Fixed with fink/fink@8c60618.

**Comment by dak180** [comment]
wrong issue

**Comment by dmacks** [comment]
Fixed via #987
